docs/guide-installation.md

```diff
@@ -169,27 +105,14 @@ gyp ERR! stack Error: `make` failed with exit code: 2
 
 ### Ubuntu/Debian Linux
 
-The best way to install any version of Node.js is to use the [NodeSource Node.js binary distributions](https://github.com/nodesource/distributions#installation-instructions). Older versions of Ubuntu install Node.js with the wrong version and binary name. If your Node binary is `nodejs` instead of `node`, or if your Node version is [`v0.10.29`](https://github.com/fivdi/onoff/wiki/Node.js-v0.10.29-and-native-addons-on-the-Raspberry-Pi), then you should follow these instructions.
-
-You'll need the package `build-essential` to compile `serialport`. If there's a binary for your platform, you won't need it. Keep rocking!
-
-```
-# Using Ubuntu and Node 6
-curl -sL https://deb.nodesource.com/setup_7.x | sudo -E bash -
-sudo apt-get install -y nodejs
-
-# Using Debian and Node 6 as root
-curl -sL https://deb.nodesource.com/setup_7.x | bash -
-apt-get install -y nodejs
-```
+You'll need the package `build-essential` to compile `serialport` if the provided binaries don't work for your system.
 
 ### Windows
-Node-Serialport supports Windows 7, 8.1, 10, and 10 IoT. Precompiled binaries are available, but if you want to build it from source you'll need to follow the [node-gyp installation](https://github.com/nodejs/node-gyp#installation) instructions. Once you've got things working, you can install Node-Serialport from source with:
+
+Node-Serialport supports Windows 10+. Precompiled binaries are available, but if you want to build it from source you'll need to follow the [node-gyp installation](https://github.com/nodejs/node-gyp#installation) instructions. Once you've got things working, you can install Node-Serialport from source with:
 
 ```powershell
-npm install serialport --build-from-source
+npm install serialport
 ```
 
 Node-gyp's documentation doesn't mention it, but it sometimes helps to create a C++ project in [Visual Studio](https://www.visualstudio.com/) so that it will install any necessary components not already installed during the past two hours of setup. This will solve some instances of `Failed to locate: "CL.exe"`.
```

Comment:
Not sure if this is still the case.  I think the biggest PITA with VS is that MSFT installers dont setup any environment variables, which isn't helpful for newbies who want to compile from the command line.

I understand that Node.js for Windows installer now automatically installs the required tools, so things should now be simpler (per https://github.com/felixrieseberg/windows-build-tools)